### PR TITLE
EIC 3D Globe v2

### PR DESCRIPTION
**Pull request by dzole0311**
The PR introduces a few changes for the EIC interactive app:

1. Introduce interactive geodesic buffer on the 3D globe as a way to pick points of interest
2. Simplify the layout of the app, as per the designs from @faustoperez 
3. Chart adjustments, code clean up

### CODE REVIEW

File: src/components/Map.jsx [hunk 1]
```diff
@@ -90,33 +81,128 @@ export default function Home() {
...
+    const { bufferLayer, pointLayer } = initializeLayers();
+
+    const createBuffer = async (point) => {
+      point.hasZ = false;
+      point.z = undefined;
+
+      const buffer = await geometryEngineAsync.geodesicBuffer(point, 560, 'kilometers');
+
+      if (pointLayer.graphics.length === 0) {
+        pointLayer.add(new Graphic({ geometry: point, symbol: pointSymbol, attributes: { name: 'Geodesic-Buffer' }   }));
```
Comment (slesaad) [inline src/components/Map.jsx:125]:
```suggestion
        pointLayer.add(new Graphic({ geometry: point, symbol: pointSymbol, attributes: { name: 'Geodesic-Point' }   }));
```

File: src/components/Panel.jsx [hunk 2]
```diff
@@ -1,30 +1,108 @@
...
+        });
+
+        const layer = mapView.map.layers.find(layer => layer.title === item.name);
+
+        if (layer) {
+            layer.visible = true;
+
+            const currentProduct = config.find(product => product.name === layer.title);
+            setCurrentJSON(currentProduct);
+        }
```
Comment (slesaad) [inline src/components/Panel.jsx:51]: can't we combine these into a single conditional?

File: src/components/Vitals.jsx [hunk 3]
```diff
@@ -1,227 +1,29 @@
...
-            setVitals(currentJSON.vitals);
-            setLayerName({ layerName: currentJSON.name, layerUnit: currentJSON.unit });
-            addSketchLayer();
-        } else {
-            removeSketchLayer(currentJSON.vitals);
-        }
-    }, [mapView, vitalSelection, currentJSON]);
+    const { vitalsData } = useContext(VitalsDataContext);
+    const globalAverage = vitalsData?.globalAverage ?? 'N/A';
+    const globalMax = vitalsData?.globalMax ?? 'N/A';
```
Comment (slesaad) [inline src/components/Vitals.jsx:7]: We're setting the default values for these in 3 different places. Is that necessary?

**Comment by slesaad** [comment]
I'm not sure how complete this PR is supposed to be, but I noticed some inconsistencies:

1. the initial view doesn't show any vitals/chart - would it make sense to show that it's loading and populate the values for the area that is selected when it's available?
2. when I have the vitals/chart on the screen for one dataset, then I click on a different dataset, it shows the same values until I drag the buffer to a different area (maybe solving 1 would solve this?)
3. might need to confirm that the stats are generated correctly (ex - in the attached image, the max value is shown as `0.13` but there are clearly values greater than `0.20` in the chart
image

**Comment by dzole0311** [comment]
Thanks for taking a look, @slesaad! You're right — it isn't a complete PR. I mainly opened it to have a deployment URL readily available. We're still refining the curated experience with @faustoperez, so some elements (and datasets) will change, and that's why I haven’t fully addressed the rest of the issues yet.

**Pull request merged**